cypress/integration/plugins/dashboards-assistant/mds_conversation_save_to_notebook_spec.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { BASE_PATH } from '../../../utils/constants';
+import { setStorageItem } from '../../../utils/plugins/dashboards-assistant/helpers';
+
+const QUESTION = 'What are the indices in my cluster?';
+const FINAL_ANSWER = 'The indices in';
+
+if (
+  Cypress.env('DASHBOARDS_ASSISTANT_ENABLED') &&
```

Comment:
Save to notebook feature will hide when MDS enabled. We only need to test this function when MDS disabled for now.